src/core/ext/filters/client_channel/xds/xds_client.cc

```diff
@@ -1746,13 +1746,24 @@ XdsClient::XdsClient(Combiner* combiner, grpc_pollset_set* interested_parties,
   }
 }
 
-XdsClient::~XdsClient() { GRPC_COMBINER_UNREF(combiner_, "xds_client"); }
+XdsClient::~XdsClient() {
+  gpr_log(GPR_INFO, "[xds_client %p] Destroying xds client", this);
```

Comment:
Please guard this by checking whether the tracer is enabled.

Reply:
Or, alternatively, just remove this line for now, and wait for me to add it as part of #21941. :)